src/compo/tempo_nc2ioda.py

```diff
@@ -41,12 +41,13 @@
 
 
 class tempo(object):
-    def __init__(self, filenames, varname, columnType, qa_flg, thin, obsVar):
+    def __init__(self, filenames, varname, columnType, qa_flg, thin, nrt, obsVar):
         self.filenames = filenames
         self.varname = varname
         self.columnType = columnType
         self.qa_flg = qa_flg
         self.thin = thin
+        self.nrt = nrt
```

Comment:
@jeromebarre I tested this out but had to update all instances of `nrt` to `v3`. Is the intent for this PR to use `v3`?

Reply:
Oh sorry about that. I learned that V3 will be reprocessed too and won't only NRT. So that's why I changed the option to` -v3`

Reply:
Okay great thanks! Yeah using `-v3` makes sense. Do you want me to update this PR or will you?